ClinGen/clincoded
# Connects to #589. Disabled curation form

Testing:

1. Create GDM, add PMID
2. Create Experimental Data object (and assess it)
3. Assess on it with another user
4. Go to Edit page with original user (should encounter disabled form due to other user assessment)
5. Hit back button to Curation Central
6. Go to Add another Experimental Data object
7. Confirm that the form is properly editable
8. Create Family object w/ Segregation data (and assess it)
9. Assess on it with another user
10. Go to Edit page with original user (should encounter un-editable Segregation section)
11. Hit back button to Curation Central
12. Click the 'Add Family' button ONCE and confirm that you go to the Add Family page (originally, you would encounter a JS error and the link would not work on first click)
